elastic/kibana
# [ResponseOps] Add telemetry for the es query rule types

Resolves https://github.com/elastic/kibana/issues/176237

## Summary

Adds new telemetry fields to track the ES Query rule search types.

### Checklist



### To verify

- Create a couple of ES Query rules with the different search types
- Create a rule that is not an ES Query rule

- Change alerting telemetry task [schedule](https://github.com/elastic/kibana/blob/main/x-pack/plugins/actions/server/usage/task.ts#L22) interval 1 min

- Run [Telemetry usage payload API](https://docs.elastic.dev/telemetry/collection/snapshot-telemetry#telemetry-usage-payload-api) in your browser console to verify the telemetry data looks the same
